packages/tokens/src/helpers/getTokensByChain.ts

```diff
@@ -0,0 +1,13 @@
+import { ChainId } from '@pancakeswap/chains'
+import { Token } from '@pancakeswap/sdk'
+
+import { allTokens } from '../allTokens'
+
+export function getTokensByChain(chainId?: ChainId): Token[] {
+  if (!chainId) {
+    return []
+  }
+
+  const tokenMap = allTokens[chainId]
```

Comment:
Although we are not tree shaking anything for now, but it will make harder to achieve. Just a thought, cannot think of any better way for now

Reply:
thought of it actually. We need to do asynchronous function call to achieve that

Reply:
another potential issue is that the gauges package is not published. So it's bundled into the smart router. Let me publish the package in a separate pr